homework/04LessonFour.js

```diff
@@ -72,7 +83,7 @@ var nestedObjects = {
       hometown: 'Omaha'
     },
     'bob roberts': {
-      starring: 'John Cusack'
+      starring: 'Tim Robbins'
```

Comment:
You need to use bracket and/or dot notation to assign the desired values in problem 9 & 10.